engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-applicable.controller.js

```diff
@@ -26,6 +30,12 @@ angular.module('Bastion.content-hosts').controller('ContentHostPackagesApplicabl
             return selected;
         };
 
+        if ($scope.katelloAgentPresent === false && $scope.remoteExecutionPresent === false) {
```

Comment:
It looks like this logic is duplicated a few times. I think it makes sense to put it as one of the attributes on the BastionConfig!

To check for `false` in this way isn't very idiomatic unless you explicitly need to check for false rather than something falsey, like undefined. It's a nitpick. I'd also write it as `hostToolingEnabled` to keep the logic framed more positively which usually keeps things easier to follow. so the pseudocode now becomes:

`hostToolingEnabled = (katelloAgentPresent || remoteExecutionPresent) ? true : false`